kubelet/datadog_checks/kubelet/cadvisor.py

```diff
@@ -149,6 +149,8 @@ def _update_container_metrics(self, instance, subcontainer, pod_list, pod_list_u
         is_pod = False
         in_static_pod = False
         subcontainer_id = subcontainer.get('id')
+        if '://' in subcontainer_id:
```

Comment:
Will `subcontainer_id` always exist? If not, this will be `if '://' in None:` which will error

Reply:
good point, fixed that ✅